src/vs/workbench/browser/parts/titlebar/media/titlebarpart.css

```diff
@@ -8,24 +8,133 @@
 	width: 100%;
 	font-size: 12px;
 	padding: 0 70px;
-	overflow: hidden;
```

Comment:
@sbatten careful, does that mean the title value can now overflow if it is very long?